src/main/java/org/trigon/hardware/phoenix6/talonfx/io/SimulationTalonFXIO.java

```diff
@@ -70,4 +66,22 @@ public void setPhysicsSimulation(MotorPhysicsSimulation physicsSimulation) {
     public TalonFX getTalonFX() {
         return talonFX;
     }
-}
+
+    /**
+     * Adapts the configuration for simulation. There are some setting that don't work well in simulation and need to be adapted.
+     * The inverted value doesn't affect the simulation, and it can sometimes cause problems so it is always set to CounterClockwise_Positive.
```

Comment:
It's not that it doesn't affect the simulation, but more that there's no reason to use it in sim